instrumentation/action_view/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-instrumentation-action_view
 
+### v0.1.1 / 2021-09-09
+
+* FIXED: Keep Active Support subscriptions intact when patching 
```

Comment:
Were some changes to instrumentation swept up unintentionally in this docs PR?

Reply:
🤔 I'll look into this. Very odd I didn't notice this was changed.

Reply:
This was due to my branch being merged instead of rebased.